docker.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Oh, sorry, this is a thing I need locally because /bin/sh is dash - I didn't mean for it to end up in the PR. I'll revert this change and keep it in a local branch